src/game/etj_timerun_v2.h

```diff
@@ -95,32 +91,33 @@ class TimerunV2 {
   void clientDisconnect(int clientNum);
   void startTimer(const std::string &runName, int clientNum,
                   const std::string &playerName, int currentTimeMs);
-  void checkpoint(const std::string &runName,
-                  int clientNum, int checkpointIndex, int currentTimeMs);
+  void checkpoint(const std::string &runName, int clientNum,
+                  int checkpointIndex, int currentTimeMs);
   void stopTimer(const std::string &runName, int clientNum, int currentTimeMs);
-  void addSeason(Timerun::AddSeasonParams season);
-  void editSeason(Timerun::EditSeasonParams params);
+  void addSeason(const Timerun::AddSeasonParams &season);
```

Comment:
😬 
All of these commands take the object and pass it to a lambda running on another thread. Replacing the actual copy done here with pass by const ref will actually cause the another thread to not use a copy but instead the original ref (which will be deleted in memory). 

**Here's ChatGPTs explanation**

In C++, when you capture a variable by reference in a lambda, the lambda holds a reference to the original variable, not a copy of it. This applies to const references as well. Here's an example:
```c++
const int x = 42;

auto lambda = [&x]() {
    std::cout << x << std::endl; // Prints 42
};

x = 43; // Error, x is const
```
The lambda above captures x by reference, so it refers to the same const int that exists outside of the lambda. Since x is const, you cannot modify it either inside or outside of the lambda.

However, it is essential to be careful with capturing by reference if the lambda might outlive the captured variable. If the captured variable goes out of scope, the reference inside the lambda will become dangling, leading to undefined behavior.

For example:
```c++
std::function<void()> createLambda() {
    const int x = 42;
    return [&x]() { std::cout << x << std::endl; }; // Warning: returning a lambda that captures a local reference
}

auto lambda = createLambda();
lambda(); // Undefined behavior, x is out of scope
```